includes/content-distribution/class-editor.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Newspack Network Content Distribution Editor.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network\Content_Distribution;
+
+use Newspack_Network\Content_Distribution;
+use Newspack_Network\Content_Distribution\Outgoing_Post;
```

Comment:
This is unused

Reply:
Whoops! If only I used phpstorm 😄 

Updated in 1fafddf4e635cad03de6fb30470d043f3555f589